docs/ruleset_functions/get_SWH_uses_associated_with_each_building_segment.md

```diff
@@ -0,0 +1,28 @@
+## get_SWH_uses_associated_with_each_building_segment
+
+Description: This function gets all the SWH uses connected to a building segment.  This function is primarily to encapsulate getting service water heating uses in one function so that if a change is made in the schema as to how service water heating use is specified, the RCT only needs to change in one place.   
```

Comment:
I see, so you are saying you would just have to update this function?

Reply:
yes